app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team):
+    return sum([member.get_rating() for member in team])
```

Comment:
```suggestion
    return sum(member.get_rating() for member in team)
```
When you write the sum like this you avoid creating a new list